flink-table/flink-sql-gateway/src/main/java/org/apache/flink/table/gateway/service/materializedtable/MaterializedTableManager.java

```diff
@@ -414,6 +422,78 @@ protected static String getManuallyRefreshStatement(
         return insertStatement.toString();
     }
 
+    private static ResultFetcher callDropMaterializedTableOperation(
+            OperationExecutor operationExecutor,
+            OperationHandle handle,
+            DropMaterializedTableOperation dropMaterializedTableOperation) {
+        ObjectIdentifier tableIdentifier = dropMaterializedTableOperation.getTableIdentifier();
+        CatalogMaterializedTable materializedTable =
+                getCatalogMaterializedTable(operationExecutor, tableIdentifier);
```

Comment:
We should consider if the table doesn't exist, but the user using `DROP MATERIALIZED TABLE IF EXISTS xxx`, nothing should happen for this case.